### DISCUSSION TITLE
Migrate to Nordlynx!!!

### DISCUSSION THREAD

**Discussion by bubuntux** [bubuntux/nordvpn#281] [General]
Hi there! 

TL;DR 
Use [nordlynx](https://github.com/bubuntux/nordlynx) container. 
Pros: 
 - Faster (like for real)
 - Do NOT depends  on nordvpn binary that create all sort of issues
Cons:
- May need to install wireguard module in the host (the container will do best effort to install if need it)
- You need to obtain the PRIVATE_KEY (you can use this container for it, check the readme)
- Still missing some ENV variables/features (still wip).

Context:
Since this container depends 100% on the nordvpn client to work makes the container unstable and unreliable, specially after updates, with the new login --nordaccount seems like this container will just stop working on newer versions, so i encourage everyone to migrate to this other container https://github.com/bubuntux/nordlynx 

The essence is the same, you use your nordvpn account (PRIVATE_KEY) to connect to the best available service and it just works, still missing a lot of features but i'll add them eventually (feel free to contribute).
I have no intention to continue maintaining this container, i just don't see the point, nordlynx would be way better in every aspect.

**Comment by imthenachoman** [comment]
Can you please elaborate on the wireguard module? If I'm running this container on unRAID, I would need to install wireguard on unRAID or in a container?
